client/js/src/helpers/api_info.ts

```diff
@@ -318,6 +318,8 @@ export function handle_message(
 					status: {
 						queue,
 						message: data.output.error as string,
+						display: data.output.display as boolean,
```

Comment:
where is `display` passed from?

Reply:
Originally had a separate display parameter. Added it back as `visible` so this makes more sense now.